x/distribution/keeper/hooks.go

```diff
@@ -107,8 +105,5 @@ func (h Hooks) BeforeValidatorModified(_ sdk.Context, _ sdk.ValAddress)
 func (h Hooks) AfterValidatorBonded(_ sdk.Context, _ sdk.ConsAddress, _ sdk.ValAddress)         {}
 func (h Hooks) AfterValidatorBeginUnbonding(_ sdk.Context, _ sdk.ConsAddress, _ sdk.ValAddress) {}
 func (h Hooks) BeforeDelegationRemoved(_ sdk.Context, _ sdk.AccAddress, _ sdk.ValAddress)       {}
-func (h Hooks) UnbondingDelegationEntryCreated(_ sdk.Context, _ sdk.AccAddress, _ sdk.ValAddress, _ int64, _ time.Time, _ sdk.Int, _ uint64) {
-}
-func (h Hooks) BeforeUnbondingDelegationEntryComplete(_ sdk.Context, _ uint64) bool {
-	return false
+func (h Hooks) AfterUnbondingOpInitiated(_ sdk.Context, _ uint64) {
```

Comment:
Why are we using `Op`? Does this mean operation? If so, we don't really need it, I think it's superfluous.

Reply:
This is from the Interchain Security spec, where I used "unbonding operation" to refer to any of the three types of unbonding operations: undelegate, redelegate, and validator unbonding. I agree that only "unbonding" could be enough. Similarly, `unbonding_id` instead  of `unbonding_op_id` should do.

Reply:
I would make that change if possible :)

Reply:
If we are ok,  I can take care of it ;)